css/values.css

```diff
@@ -0,0 +1,336 @@
+/*
+===================================
+===================================
+Our Values
+===================================
+===================================
+*/
+
+/* 
+===================================
+Section Styles (background, padding, etc.)
+===================================
+*/
+
+/* 
+===================================
+Hero
+===================================
+*/
+.hero {
+	background: linear-gradient(
+			260deg,
+			rgba(9, 9, 9, 0.8) 46.9%,
+			rgba(9, 9, 9, 0) 86.82%
+		),
+		url("../assets/images/pages/values/hero-values.png") top/cover;
+	display: flex;
+	align-items: center;
+}
+/* values heading layout */
+.hero .wrapper {
+	display: flex;
+	justify-content: flex-end;
+}
+/* values heading sizing */
+.hero-text-container {
+	width: 50%;
+	display: flex;
+	flex-direction: column;
+	gap: var(--spacing-sm);
+}
+
+/* 
+===================================
+Values
+===================================
+*/
+
+/* layout */
+.values-list,
+.values-row,
+.values-img-text {
+	display: flex;
+	gap: var(--spacing-sm);
+}
+
+.values-list,
+.values-img-text {
+	flex-direction: column;
+}
+
+/* card widths */
+.sustainability,
+.authenticity {
+	width: 60%;
+}
+
+.humanity,
+.multigenerational {
+	width: 40%;
+}
+
+.education,
+.innovation {
+	width: 50%;
+}
+
+/* general card styling */
+.values-card {
+	padding: 60px;
+	color: var(--text-color-light);
+	border-radius: var(--border-radius);
+
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+/* text layout*/
+.values-card .text {
```

Comment:
Let's limit the width of this text container so that paragraph doesn't get too long. 

<img width="831" alt="Screen Shot 2024-01-11 at 3 10 27 PM" src="https://github.com/indigenousfriends/itlct-website-development/assets/704697/d4775ab6-ff83-4fdc-a19d-e76795f28e35">

Reply:
set a max width to cards w text img  and gave them an additional **text-left** and **text-right** class to move them away from the background img. 

all reset in 1024px MQ!

Reply:
![image](https://github.com/indigenousfriends/itlct-website-development/assets/98284615/7fc624b5-e881-49da-b6c2-f0cea8784299)

Reply:
Looks much better, thanks!